dotcom-rendering/src/components/SubheadingBlockComponent.tsx

```diff
@@ -68,15 +60,23 @@ const getFontStyles = ({
 	}
 `;
 
-const getStyles = (format: ArticleFormat) => {
+const paddingStyles = css`
+	padding-top: ${space[2]}px;
+	padding-bottom: ${space[0]}px;
+	${from.tablet} {
+		padding-bottom: ${space[1]}px;
+	}
+`;
+
+export const subheadingStyles = (format: ArticleFormat) => {
```

Comment:
I think it’s best not to export these styles and rather create a generic H2 component for this purpose.

Reply:
Much better!